### PR TITLE
NME - Adding input to slider

### PR DESCRIPTION
**Pull request by msDestiny14**
slider

https://github.com/BabylonJS/Babylon.js/issues/9159

### CODE REVIEW

File: nodeEditor/src/sharedComponents/sliderLineComponent.tsx [hunk 1]
```diff
@@ -100,14 +103,26 @@ export class SliderLineComponent extends React.Component<ISliderLineComponentPro
     }
 
     render() {
+
+        const input = this.props.globalState ?
+        <FloatLineComponent globalState={this.props.globalState} smallUI={true} label="" target={this.state} propertyName="value" min={this.prepareDataToRead(this.props.minimum)} max={this.prepareDataToRead(this.props.maximum)}
```
Comment (deltakosh) [inline nodeEditor/src/sharedComponents/sliderLineComponent.tsx:108]: why not having the input all the time?

Comment (msDestiny14) [reply nodeEditor/src/sharedComponents/sliderLineComponent.tsx:108]: do you think we should? I'm not sure all the places that include it are wanted but I can check.

Comment (deltakosh) [reply nodeEditor/src/sharedComponents/sliderLineComponent.tsx:108]: I think so unless we find an obvious reason not to do so (same for inspector by the way)

**Comment by deltakosh** [comment]
Ok excellent please do the same for the inspector :)

**Pull request merged**